.github/workflows/auto-updater.yml

```
@@ -0,0 +1,46 @@
+name: Automatic Data Update
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 2 * * *'
+
+jobs:
+  DataUpdater:
+    name: Automatic Data Update
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'sapphiredev'
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@v4
+      - name: Use Node.js v20
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: yarn
+          registry-url: https://registry.yarnpkg.com/
+      - name: Install Dependencies
+        run: yarn --immutable
```

```suggestion
      - name: Install dependencies
        uses: sapphiredev/.github/actions/install-yarn-dependencies@main
        with:
          node-version: 20
```

Using reusable actions